src/components/date-picker/z-range-picker/index.tsx

```diff
@@ -0,0 +1,595 @@
+import {
+  Component,
+  Prop,
+  Element,
+  h,
+  Listen,
+  EventEmitter,
+  Event,
+  State,
+} from "@stencil/core";
+
+import flatpickr from "flatpickr";
+import { Italian } from "flatpickr/dist/l10n/it.js";
+
+import {
+  ZDatePickerMode,
+  ZDatePickerModeValues,
+  ZDatePickerPosition,
+  RangePickerActiveInput,
+} from "../../../beans";
+
```

Comment:
qui andrebbe aggiunta la jsdoc, documentando la variabile css `--toggle-height` (anche se non ho capito a che serve). poi le css vars dei componenti è meglio prefissarle con il nome del componente, tipo `--z-range-picker--toggle-height` per evitare conflitti di nome, non si sa mai

```suggestion

/**
 * Range picker component.
 * 
 * @cssprop --toggle-height - descrizione della variabile
 */
```